dashboard/app/controllers/user_level_evaluations_controller.rb

```diff
@@ -0,0 +1,35 @@
+require 'json'
+
+class UserLevelEvaluationsController < ApplicationController
+  include LevelsHelper
+  include Rails.application.routes.url_helpers
+  before_action :authenticate_user!
+  load_and_authorize_resource :user_level_interaction
+
+  # POST /user_level_evaluations
+  def create
```

Comment:
please add unit tests for who can access this method. I would suggest that a user should only be able to create an entry for themselves or one of their students, such as we do here:
* https://github.com/code-dot-org/code-dot-org/blob/601210b30e249e4677024876686ca5b0d825541c/dashboard/app/models/ability.rb#L178-L179
* https://github.com/code-dot-org/code-dot-org/blob/601210b30e249e4677024876686ca5b0d825541c/dashboard/app/models/ability.rb#L183-L184

Reply:
thanks for the examples!